mitodl/hugo-course-publisher
# build external content

#### What are the relevant tickets?
https://github.com/mitodl/hugo-course-publisher/issues/191

#### What's this PR do?
Adds the `build:externalcontent` command to `package.json`.

#### How should this be manually tested?
Use [this](https://github.com/mitodl/ocw-to-hugo/tree/cg/content-dir) branch of `ocw-to-hugo` to generate markdown for any number of courses, just the example ones are fine.  Then on this branch of `hugo-course-publisher`, run the following:

```sh
npm run build:external -- /path/to/ocw-to-hugo/output
```

When complete, you should have a fully usable site in the `dist` folder with all the content from the folder you pointed it to.
